cql2pgjson/src/main/java/org/folio/cql2pgjson/CQL2PgJSON.java

```diff
@@ -168,15 +171,19 @@ public void setDbSchemaPath(String dbSchemaPath) {
   }
 
   private void loadDbSchema(String schemaPath) {
+    if (schemaPath == null) {
+      schemaPath = "templates/db_scripts/schema.json";
+    }
+    dbSchema = dbSchemaCache.computeIfAbsent(schemaPath, CQL2PgJSON::loadDbSchemaFromResource);
+  }
+
+  private static Schema loadDbSchemaFromResource(String schemaPath) {
     try {
-      if (schemaPath == null) {
-        schemaPath = "templates/db_scripts/schema.json";
-      }
       String dbJson = ResourceUtil.asString(schemaPath, CQL2PgJSON.class);
       logger.info("loadDbSchema: Loaded {} OK", schemaPath);
-      dbSchema = ObjectMapperTool.getMapper().readValue(dbJson, Schema.class);
-    } catch (IOException ex) {
-      logger.error("No schema.json found", ex);
+      return ObjectMapperTool.getMapper().readValue(dbJson, Schema.class);
+    } catch (IOException e) {
+      throw new UncheckedIOException(e);
```

Comment:
Since there is no logging here, i think the exception will get swallowed in the `dbSchemaCache.computeIfAbsent` call above since the call is effectively in a lambda. Do you agree?

Reply:
Yep, I saw that earlier. Looks like setDbSchemaPath calls the method. All is well!